### PR TITLE
Update getting-started-macos.md

### PR DESCRIPTION
**Pull request by evilnick**

### CODE REVIEW

**Review by alfsuse — APPROVED** [review]

**Comment by alfsuse** [comment]
same though as  https://github.com/kubeflow/website/pull/2273

**Review by alfsuse — APPROVED** [review]

File: content/en/docs/started/workstation/getting-started-macos.md [hunk 1]
```diff
@@ -4,44 +4,37 @@ description = "Install Kubeflow on macOS"
...
+- Install [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/)
+
+Follow the instructions on [Kubeflow on MiniKube](/docs/started/workstation/minikube-linux/) to complete this path.
 
-The instructions on [Multipass and MicroK8s getting started](/docs/started/workstation/getting-started-multipass/)
-page will complete this path.
+### Multipass
 
-## Kubernetes appliance
+[Multipass](https://multipass.run/) creates a Linux virtual machine on Windows, Mac or Linux systems. The VM contains a complete Ubuntu operating
```
Comment (RFMVasconcelos) [inline content/en/docs/started/workstation/getting-started-macos.md:35]: Mac -> macOS

File: content/en/docs/started/workstation/getting-started-macos.md [hunk 2]
```diff
@@ -4,44 +4,37 @@ description = "Install Kubeflow on macOS"
...
 from fully-assembled Kubeflow stacks, to stacks that require some assembly.
 
-## Kubeflow appliance
+*Note:* It is recommended that you have at least 12GB of RAM and 50GB of storage available.
 
-A Kubeflow appliance is a virtual machine that has Kubeflow already installed. Once the
-necessary supporting software is installed no further installation steps are required.
+### MicroK8s
 
-### MiniKF
```
Comment (RFMVasconcelos) [inline content/en/docs/started/workstation/getting-started-macos.md:20]: You have deleted MiniKF, please undo

File: content/en/docs/started/workstation/getting-started-macos.md [hunk 3]
```diff
@@ -4,44 +4,37 @@ description = "Install Kubeflow on macOS"
...
+MicroK8s is a lightweight zero-ops Kubernetes which has a native installer for macOS. MicroK8s is highly available from 3+ nodes and includes a single-command install of Kubeflow.
 
-MiniKF is a predefined virtual machine that installs onto VirtualBox through Vagrant.
-The only following applications are required to use MiniKF:
+To get Kubeflow:
 
-- Install [Vagrant](https://www.vagrantup.com/downloads.html)
-- Install [Virtual Box](https://www.virtualbox.org/wiki/Downloads)
+1. Install [MicroK8s](https://microk8s.io/) - see the "Alternative Install" page for links to the latest macOS installer.
```
Comment (RFMVasconcelos) [inline content/en/docs/started/workstation/getting-started-macos.md:19]: `see the "Alternative Install" page for links to the latest macOS installer.`
 
This does not seem to be needed, as the installer for macOS is in the main MicroK8s page.

**Comment by RFMVasconcelos** [comment]
Thank you for restructuring this @evilnick !

@alfsuse, thank you for your offer! :)
For the moment I think the restructuring that @evilnick did is already a great step forward as this doc has been outdated and messy for quite a while. I propose we close this PR to have these changes take effect and then potentially evaluate adding more complexity in other PRs.

**Pull request merged**